elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/RealmDefinition.java

```diff
@@ -28,6 +33,9 @@
  */
 class RealmDefinition extends SimpleResourceDefinition {
 
+    static final ResourceRegistration PATH = ResourceRegistration.of(PathElement.pathElement(ElytronOidcDescriptionConstants.REALM), Stability.DEFAULT);
+    static final SimpleAttributeDefinition TYPE = new SimpleAttributeDefinitionBuilder("type", ModelType.STRING)
```

Comment:
Just to check, where is TYPE being used?

Reply:
I am not using it anymore. 
I have removed it. 
Thanks for catching it!